Applications/Games/Total War Rome II/Steam/script.js

```diff
@@ -22,7 +22,6 @@ new SteamScript()
         wizard.message(
             tr(
                 "If you are experiencing issues with game (e.g. it crashes at start or rendering is broken), you can try to enable de OpenGL renderer, by modifying :\n\n gfx_device_type to 2\n\n in the {0}/drive_c/users/USERNAME/Application Data/The Creative Assembly/Rome2/scripts/preferences_script.txt",
```

Comment:
This cannot work. The "{0}" in the `tr` expects a parameter.

Reply:
Can I use `wine.prefixDirectory()` then?

Reply:
Yes. #1179 is basically about the missing `()` at the end